spec/tapioca/cli_spec.rb

```diff
@@ -0,0 +1,30 @@
+# typed: true
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Tapioca
+  class CliSpec < SpecWithProject
+    describe "Tapioca::Cli" do
+      before do
+        @command_stub = mock(run: nil)
+      end
+
+      it "passes through the `lsp_addon` flag to the DslGenerate command" do
+        Commands::DslGenerate.expects(:new).with do |options|
+          options[:lsp_addon] == true
+        end.returns(@command_stub)
+
+        capture_io { Cli.start(["dsl", "--lsp_addon"]) }
```

Comment:
Do we still print things even with the stub? I would've expected nothing to be printed given that we're stubbing `run` to do nothing.

Reply:
In the second test is suppresses `Unknown switches "--another-flag"`. I can remove it from the first.